doc/source/ray-core/objects/serialization.rst

```diff
@@ -202,6 +202,61 @@ There are at least 3 ways to define your custom serialization process:
      except TypeError:
         pass
 
```

Comment:
Create a link here by doing

```
.. _custom-exception-serializer
```

so that later on you can refer in the docstring.